src/react/components/Highlight.jsx

```diff
@@ -0,0 +1,35 @@
+/*
+   This file is part of Astarte.
+
+   Copyright 2020 Ispirata Srl
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+import React from 'react';
+
+const Highlight = ({ text, word }) => {
+  if (!word) {
+    return text;
+  }
+
+  return text.split(word).reduce((prev, current, index) => [
+    prev,
+    <span key={index} className="bg-warning text-dark">
+      {word}
+    </span>,
+    current,
+  ]);
+};
```

Comment:
I think it would be cool to use this component passing the text as `children` instead of via props.

```jsx
<Highlight text={"words to highlight"}>
  Some random text that includes the words to highlight
</Highlight>
```

Reply:
that's a nice idea!